speakeasy/winenv/api/usermode/kernel32.py

```diff
@@ -5411,3 +5438,70 @@ def Wow64RevertWow64FsRedirection(self, emu, argv, ctx={}):
         rv = 1
 
         return rv
+
+    @apihook('EnumProcesses', argc=3)
+    def EnumProcesses(self, emu, argv, ctx={}):
+        '''
+        BOOL EnumProcesses(
+          DWORD   *lpidProcess,
+          DWORD   cb,
+          LPDWORD lpcbNeeded
+        );
+        '''
+        lpidProcess, cb, lpcbNeeded = argv
+        processes = emu.get_processes()
+
+        lpidProcess_cursor = lpidProcess
+        lim = min(cb / 4, len(processes))
```

Comment:
```suggestion
        lim = min(cb // 4, len(processes))
```

nit: integer division